hikari/impl/rest.py

```diff
@@ -1078,6 +1078,78 @@ async def edit_permission_overwrites(
         body.put("deny", deny)
         await self._request(route, json=body, reason=reason)
 
+    @deprecation.deprecated("2.0.0.dev110", "edit_permission_overwrite")
+    async def edit_permission_overwrites(
+        self,
+        channel: snowflakes.SnowflakeishOr[channels_.GuildChannel],
+        target: typing.Union[
+            snowflakes.Snowflakeish, users.PartialUser, guilds.PartialRole, channels_.PermissionOverwrite
+        ],
+        *,
+        target_type: undefined.UndefinedOr[typing.Union[channels_.PermissionOverwriteType, int]] = undefined.UNDEFINED,
+        allow: undefined.UndefinedOr[permissions_.Permissions] = undefined.UNDEFINED,
+        deny: undefined.UndefinedOr[permissions_.Permissions] = undefined.UNDEFINED,
+        reason: undefined.UndefinedOr[str] = undefined.UNDEFINED,
+    ) -> None:
+        """Edit permissions for a specific entity in the given guild channel.
+
```

Comment:
This doc string doesn't really need to be this long considering the only important info is that this is a deprecated alias of `edit_permission_overwrites`